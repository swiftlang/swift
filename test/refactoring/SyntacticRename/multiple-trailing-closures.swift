func /*test:def*/test(x: () -> (), _ xx: () -> (), y: () -> (), z: ()->()) {}

/*test:call*/test(x: {}, {}, y: {}) {}
/*test:call*/test(x: {}, {}) {} z: {}
/*test:call*/test(x: {}) {} y: {} z: {}
/*test:call*/test {} _: {} y: {} z: {}


// REQUIRES: swift_swift_parser
// RUN: %empty-directory(%t.ranges)
// RUN: %refactor -find-rename-ranges -source-filename %s -pos="test" -is-function-like -old-name "test(x:_:y:z:)" >> %t.ranges/multiple-trailing-closures.swift.expected
// RUN: diff -u %S/Outputs/multiple-trailing-closures.swift.expected %t.ranges/multiple-trailing-closures.swift.expected
