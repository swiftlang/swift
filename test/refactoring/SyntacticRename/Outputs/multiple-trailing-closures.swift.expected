func /*test:def*/<base>test</base>(<arglabel index=0>x</arglabel><param index=0></param>: () -> (), <arglabel index=1>_</arglabel><param index=1> xx</param>: () -> (), <arglabel index=2>y</arglabel><param index=2></param>: () -> (), <arglabel index=3>z</arglabel><param index=3></param>: ()->()) {}

/*test:call*/<base>test</base>(<callarg index=0>x</callarg><callcolon index=0>: </callcolon>{}, <callcombo index=1></callcombo>{}, <callarg index=2>y</callarg><callcolon index=2>: </callcolon>{}) {}
/*test:call*/<base>test</base>(<callarg index=0>x</callarg><callcolon index=0>: </callcolon>{}, <callcombo index=1></callcombo>{}) {} <sel index=3>z</sel>: {}
/*test:call*/<base>test</base>(<callarg index=0>x</callarg><callcolon index=0>: </callcolon>{}) {} <sel index=2>y</sel>: {} <sel index=3>z</sel>: {}
/*test:call*/<base>test</base> {} <sel index=1>_</sel>: {} <sel index=2>y</sel>: {} <sel index=3>z</sel>: {}


